MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -2138,6 +2141,11 @@ private string GetFixedSizeScalingFilter(Video3DFormat? threedFormat, int reques
                     parameters += ":range={5}";
                 }
 
+                if (string.Equals(GetHwaccelType(state, options, state.OutputVideoCodec), "-hwaccel opencl", StringComparison.OrdinalIgnoreCase))
+                {
+                    filters.Add("format=p010");
```

Comment:
No need to add p010 conversion if the decoder is hardware decoder(NVDEC or D3D11VA or AMF decoder in the future).